apps/src/lab2/levelEditors/aiCustomizations/CollapsibleSection.tsx

```diff
@@ -0,0 +1,33 @@
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+import {BodyOneText} from '@cdo/apps/componentLibrary/typography';
+import React, {useState} from 'react';
+import moduleStyles from './edit-ai-customizations.module.scss';
+
+const CollapsibleSection: React.FunctionComponent<{
+  title: string;
+  children: React.ReactNode;
+  initiallyCollapsed?: boolean;
+}> = ({title, children, initiallyCollapsed = true}) => {
+  const [collapsed, setCollapsed] = useState(initiallyCollapsed);
+
+  return (
+    <div>
```

Comment:
nit: Would this be a good place to use a React.Fragment (`<>`) instead of `<div>` to avoid extra bloat in the HTML?

Reply:
Oh good call! Will do